pocs/core.py

```diff
@@ -38,7 +38,12 @@ class POCS(PanStateMachine, PanBase):
 
     """
 
-    def __init__(self, state_machine_file='simple_state_table', messaging=False, **kwargs):
+    def __init__(
+            self,
+            observatory,
+            state_machine_file='simple_state_table',
+            messaging=False,
+            **kwargs):
 
         # Explicitly call the base classes in the order we want
```

Comment:
FWIW, this comment and the associated code seem to imply that we should use aggregation for PanStateMachine.

Reply:
I think this might be the comment that you missed, about the awkwardness of multiple inheritance.